### DISCUSSION TITLE
Exporting to humdrum **kern

### DISCUSSION THREAD

**Discussion by eneugut** [CPJKU/partitura#198] [General]
Hi! I saw in the Tutorial that support for humdrum **kern is coming. I'm wondering if you have a timeline for when it will be released? 

Alternatively, do you know of any tools for converting MusicXML files to **kern? I'm having issues using the Humdrum Toolkit, and music21 does not seem to support this functionality. 

Thanks so much -- this library is great.

**Comment by manoskary** [comment]
Hello, @eneugut Kern functionality is partially supported in partitura. You can import Kern files using partitura just like you import musicxml, midi or mei files. You can use the command:

```python
partitura.load_kern(path/to/file)
```
Of course, there are always things to add and to correct so do not hesitate to reach out if you have any issues.

Currently, partitura can import Kern but cannot export Kern. The export Kern functionality is not something we plan for now. 

What kind of music are you working with, i.e. mensural, renaissance, classical, pop?

**Reply by eneugut** [reply]
Ok understood. 

I'm working with contemporary music -- pop, jazz, folk, rock, etc.
